cpp/tests/groupby/group_sum_test.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2020, NVIDIA CORPORATION.
```

Comment:
in few files, year is updated, in few, it's not.
what are the rules for updating the year?

Reply:
If you change a file, make sure the current year is included. Since this PR was started in 2020, most files didn't need a year update.